docs/src/pages/component-demos/grid-list/ImageGridList.js

```diff
@@ -0,0 +1,118 @@
+// @flow weak
+
+import React from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import { GridList, GridTile } from 'material-ui/GridList';
+
+import breakfastImage from 'docs/src/assets/images/grid-list/00-52-29-429_640.jpg';
+import burgerImage from 'docs/src/assets/images/grid-list/burgers-1976198_640.jpg';
+import cameraImage from 'docs/src/assets/images/grid-list/camera-813814_640.jpg';
+import morningImage from 'docs/src/assets/images/grid-list/morning-819362_640.jpg';
+import hatsImage from 'docs/src/assets/images/grid-list/hats-829509_640.jpg';
+import honeyImage from 'docs/src/assets/images/grid-list/honey-823614_640.jpg';
+import vegetableImage from 'docs/src/assets/images/grid-list/vegetables-790022_640.jpg';
+import plantImage from 'docs/src/assets/images/grid-list/water-plant-821293_640.jpg';
+import mushroomImage from 'docs/src/assets/images/grid-list/fly-21685_640.jpg';
+import oilImage from 'docs/src/assets/images/grid-list/olive-oil-1412361_640.jpg';
+import seastarImage from 'docs/src/assets/images/grid-list/sea-star-1501698_640.jpg';
+import bikeImage from 'docs/src/assets/images/grid-list/bike-190483_640.jpg';
+
+const styleSheet = createStyleSheet('ImageGridList', () => ({
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'space-around',
+    backgroundColor: 'white',
+  },
+  gridList: {
+    width: 500,
+    height: 450,
+    overflowY: 'auto',
+  },
+  subheader: {
+    width: '100%',
+  },
+}));
+
+const tilesData = [
```

Comment:
This array is repeated over the 4 demos. What about extracting it in a module? The demos are already not working out of the box because poeple needs to import the images.